pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -107,10 +107,11 @@ def test_write_append_mode(ext, mode, expected):
             assert wb2.worksheets[index]["A1"].value == cell_value
 
 
-def test_to_excel_with_openpyxl_engine(ext, tmpdir):
+@pytest.mark.parametrize("col", ["B", "render"])
+def test_to_excel_with_openpyxl_engine(ext, tmpdir, col):
     # GH 29854
     df1 = DataFrame({"A": np.linspace(1, 10, 10)})
-    df2 = DataFrame({"B": np.linspace(1, 20, 10)})
+    df2 = DataFrame({col: np.linspace(1, 20, 10)})
```

Comment:
does this actually test anything? i would simply paraterize df1

Reply:
Took an old test because I wanted to avoid unnecessary code duplication and parametrized it. The test was previously added because of an openpyxl issue. I could revert the parametrization and add an own test, but I would like to avoid changing a previously existing test?

Reply:
you are welcome to add a new test

Reply:
I changed the old one back and added a new one